src/sentry/static/sentry/app/views/alerts/rules/details/metricChart.tsx

```diff
@@ -129,7 +129,7 @@ function createIncidentSeries(
         `<div class="tooltip-series"><div>`,
         `<span class="tooltip-label">${marker} <strong>${t(
           'Alert'
-        )} #${identifier}</strong></span>${seriesName} ${dataPoint?.value}`,
+        )} #${identifier}</strong></span>${seriesName} ${dataPoint?.value?.toLocaleString()}`,
```

Comment:
does `toFixed` make more sense?

Reply:
toLocaleString kinda handles that for us while adding commas and other nice things